src/lib/RequestParser.js

```diff
@@ -70,6 +70,9 @@ class RequestParser { // eslint-disable-line no-unused-vars
             if (!allowEmpty) throw new Errors.InvalidRequestError('Label must not be empty');
             return undefined;
         }
+        if (Utf8Tools.stringToUtf8ByteArray(label).length > 63) {
```

Comment:
```suggestion
        if (Utf8Tools.stringToUtf8ByteArray(label).byteLength > 63) {
```

Reply:
changed it in `parseTransaction`, too. Is there a real difference, though?